applications/poisson/sine/tests/curvilinear_AMG.output

```diff
@@ -0,0 +1,219 @@
+
```

Comment:
Doesn't this file miss `.with_trilinos=true`?

Reply:
yes :man_facepalming: